embeddings/bigquery-import.ipynb

```diff
@@ -0,0 +1,354 @@
+        "(1, [0.1, 0.1, 0.1], \"allow1\", \"deny1\", 1, 0.1, \"metadata1\"),\n",
+        "(2, [0.2, 0.2, 0.2], \"allow2\", \"deny2\", 2, 0.2, \"metadata2\");\n",
+        "SELECT * FROM import_example_dataset.test_table;"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7a2d7f3e3a33"
+      },
+      "source": [
+        "## Create a Vector Search Index\n",
+        "\n",
+        "This may take a few moments to finish the index creation."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "10kzx4YoVjMU"
+      },
+      "outputs": [],
+      "source": [
+        "my_index = aiplatform.MatchingEngineIndex.create_tree_ah_index(\n",
+        "    display_name='import_test_index_name',\n",
+        "    dimensions=3,\n",
+        "    approximate_neighbors_count=10,\n",
+        "    index_update_method=\"BATCH_UPDATE\",\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "dvqIfG421PW7"
+      },
+      "source": [
+        "## Import from BigQuery into Vector Search\n",
+        "\n",
+        "This returns a Long-Running Operation (LRO), which allows you to track the progress of the import operation."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import requests\n",
+        "\n",
+        "# Get token to use for REST request\n",
+        "gcloud_token = !gcloud auth print-access-token\n",
+        "\n",
+        "url = f\"https://{LOCATION}-aiplatform.googleapis.com/v1beta1/{my_index.resource_name}:import\"\n",
+        "headers = {\n",
+        "    \"Content-Type\": \"application/json\",\n",
+        "    \"Authorization\": f\"Bearer {gcloud_token[0]}\"\n",
+        "}\n",
+        "request = {\n",
+        "  \"is_complete_overwrite\": True,\n",
+        "  \"config\": {\n",
+        "    \"big_query_source_config\": {\n",
+        "      \"table_path\": f\"bq://{PROJECT_ID}.import_example_dataset.test_table\",\n",
+        "      \"datapoint_field_mapping\": {\n",
+        "        \"id_column\": \"id\",\n",
+        "        \"embedding_column\": \"embedding\",\n",
+        "        \"restricts\": [\n",
+        "          {\"namespace\": \"restrict\", \"allow_column\": [\"allow_column\"], \"deny_column\": [\"deny_column\"]},\n",
+        "        ],\n",
+        "        \"numeric_restricts\": [\n",
+        "          {\"namespace\": \"int_restrict\", \"value_column\": \"int_column\", \"value_type\": \"INT\"},\n",
+        "          {\"namespace\": \"float_restrict\", \"value_column\": \"float_column\", \"value_type\": \"FLOAT\"}\n",
+        "        ]\n",
+        "        # The import may include metadata if your project has been allow-listed\n",
+        "        # for the VS metadata preview.\n",
+        "        #\"embedding_metadata\": \"metadata_column\"\n",
+        "      }\n",
```

Comment:
Can you change this to use the Vertex AI SDK/GenAI SDK/Client Library?

Reply:
Thanks, yes, I will update this to use the SDK once I get cl/775528838 (https://github.com/googleapis/python-aiplatform/pull/5473) merged in and the SDK has support for this feature.

Reply:
Ok, thanks. I'll approve/merge, then you can update it once the SDK is rolled out.